lib/internal/modules/cjs/loader.js

```diff
@@ -1293,5 +1294,10 @@ Module.syncBuiltinESMExports = function syncBuiltinESMExports() {
   }
 };
 
+Module.isBuiltIn = function isBuiltIn(moduleName) {
+  moduleName = StringPrototypeReplace(String(moduleName), /^node:/, '');
+  return ArrayPrototypeIncludes(Module.builtinModules, moduleName);
```

Comment:
i think it's still a perf hit to reassign to function arguments
```suggestion
  return ArrayPrototypeIncludes(
    Module.builtinModules,
    StringPrototypeReplace(String(moduleName), /^node:/, '')
  );
```